### PR TITLE
Update pulp-fiction

### PR DESCRIPTION
**Pull request by makkoncept**

### CODE REVIEW

File: pulp-fiction [hunk 1]
```diff
@@ -1 +1,3 @@
 best of Tarantino? 
+
```
Comment (cold-hotdog) [inline pulp-fiction:2]: review comment

**Review by ghost — CHANGES_REQUESTED** [review]
request-changes

**Review by cold-hotdog — CHANGES_REQUESTED** [review]
req

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
dd

**Review by cold-hotdog — CHANGES_REQUESTED** [review]
qwer

**Review by ghost — APPROVED** [review]

**Review by cold-hotdog — APPROVED** [review]

**Review by cold-hotdog — CHANGES_REQUESTED** [review]
qwerty

**Pull request closed without merging**